### PR TITLE
Add LICENSE

### PR DESCRIPTION
**Pull request by gwynne**

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Contributing
+
+We're excited to see you're interested in contributing to the AsyncKit package! This document covers how you can report any issues you find or contribute with bug fixes and new features!
```
Comment (0xTim) [inline .github/CONTRIBUTING.md]:
```suggestion
We're excited to see you're interested in contributing to the ReleaseBot! This document covers how you can report any issues you find or contribute with bug fixes and new features!
```

**Review by 0xTim — CHANGES_REQUESTED** [review]
AsyncKit leftover, though I'm just going to check to see if we can use our central contributing

**Comment by 0xTim** [comment]
I propose we stick the contributing file in https://github.com/vapor/.github, as described by https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file then we have a central one we can update

**Comment by gwynne** [comment]
Moved to https://github.com/vapor/.github/pull/2

**Review by 0xTim — APPROVED** [review]

**Pull request merged**